example/grpc/README.md

```diff
@@ -0,0 +1,26 @@
+# gRPC Tracing Example
+
+Traces client and server calls via interceptors.
+
+### Compile .proto
```

Comment:
We should clarify that recompiling the protobufs is only necessary if you're changing them.